dynatrace-activegate-operator-google-marketplace/charts/dynatrace-activegate-operator/templates/serviceaccount-operator.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "dynatrace-activegate-operator.labels" . | nindent 4 }}
+imagePullSecrets:
+  - name: dynatrace-activegate-registry
```

Comment:
Do we need this?

Reply:
yes, for our new update mechanism

Reply:
Any reason to have it here instead of the Pod?